spec/support/json_validator.rb

```diff
@@ -0,0 +1,8 @@
+RSpec::Matchers.define :match_response_schema do |schema|
+  match do |response|
+    schema_directory = "#{Dir.pwd}/spec/support/api/schemas"
```

Comment:
Not sure if the `...support/api/schemas` is necessary as the app is an API only app. I think this wouldn't be less meaningful: `...support/schemas` What do you think @andela-oakinniranye?